BookingPageController.cls

```diff
@@ -0,0 +1,31 @@
+public with sharing class BookingPageController {
```

Comment:
Extention should be end with keyword extention and controller should end with controller.